discovery-infra/day2.py

```diff
@@ -0,0 +1,106 @@
+import os
+import utils
+import waiting
+import uuid
+import assisted_service_api
+import consts
+
+
+def execute_day2_flow(internal_cluster_name, num_workers, api_vip_ip, api_vip_dnsname):
+    utils.recreate_folder(consts.IMAGE_FOLDER, force_recreate=False)
```

Comment:
you created an image folder at main of start_discovery.py, I think we should receive it as an argument